packages/flutter_localizations/lib/src/l10n/generated_material_localizations.dart

```diff
@@ -449,6 +452,9 @@ class MaterialLocalizationAm extends GlobalMaterialLocalizations {
   @override
   String get licensesPackageDetailTextOther => '\$licenseCount licenses';
 
+  @override
```

Comment:
Did you hand modify this file? It is a file that we generate from the *.arb files with the `gen_localizations.dart` script. See comment at the top of `material_localizations.dart' for instructions on how to use it.

Reply:
No, I didn't hand modify this.